src/main/kotlin/racing/domain/Cars.kt

```diff
@@ -0,0 +1,23 @@
+package racing.domain
+
+import racing.domain.strategy.MovingStrategy
+import racing.domain.strategy.RandomMovingStrategy
+
+object Cars {
+    const val NAME_SEPARATOR = ","
+
+    fun createCars(nameOfCars: String, movingStrategy: MovingStrategy = RandomMovingStrategy): List<Car> {
+        val nameList = nameOfCars.split(NAME_SEPARATOR)
+        val cars = mutableListOf<Car>()
+        nameList.onEach {
+            cars.add(Car(it, movingStrategy))
+        }
```

Comment:
```suggestion
        val cars = nameList.map { Car(it, movingStrategy) }
```
이렇게 표현하면 `cars`를 불변하게 사용할 수 있어요 :)